src/pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -451,13 +451,12 @@ def on_predict_batch_end(self, trainer, pl_module, outputs, batch, batch_idx, da
 
     def _get_train_description(self, current_epoch: int) -> str:
         train_description = f"Epoch {current_epoch}"
+        if self.trainer.max_epochs is not None:
+            train_description += f"/{self.trainer.max_epochs - 1}"
         if len(self.validation_description) > len(train_description):
             # Padding is required to avoid flickering due of uneven lengths of "Epoch X"
             # and "Validation" Bar description
-            num_digits = len(str(current_epoch))
-            required_padding = (len(self.validation_description) - len(train_description) + 1) - num_digits
-            for _ in range(required_padding):
-                train_description += " "
+            train_description = f"{train_description:{len(self.validation_description)}}"
```

Comment:
does this need to be fstring when there is just variable printing?

Reply:
@Borda Wdym?
It will be an argument to the `TextColumn` https://rich.readthedocs.io/en/stable/progress.html#columns

Reply:
```suggestion
            train_description = repr(train_description:{len(self.validation_description)})
```

Reply:
It's not just variable printing, it's using the padding argument of f-strings as indicated in the comment above.

Using `repr` here is not equivalent.